source/_integrations/accuweather.markdown

```diff
@@ -27,3 +27,9 @@ To add AccuWeather to your installation, go to **Configuration** >> **Integratio
 Due to limitations of the terms of use of AccuWeather free API key, it is possible to configure only one integration instance.
 
 </div>
+
+<div class="note warning">
+
+The Limited Trial account only allows 50 API calls per day.
+
+</div>
```

Comment:
Please combine it with the warning that is already there. We don't have to add a new warning box for this.